newdle/client/src/components/ParticipantTable.js

```diff
@@ -109,13 +115,15 @@ function AvailabilityRow({
             <div>
               <div className={styles['date']}>{startTime.format('D MMM')}</div>
               <div className={styles['time']}>{formatMeetingTime(startTime, duration)}</div>
+              <div className={styles['timezone']}>{newdleTimezone}</div>
```

Comment:
It looks good, but I wonder if it wouldn't make more sense to display the timezone in a single place instead of for every single timeslot (since it's always the same anyway).

Not a blocker for this PR though!

Reply:
I adds a lot of clutter in the case you have many slots, but I believe that it also helps people not loosing track about the original timestamp. But I do also see a use case for rendering it only once, specially because in the future we may decide to display this time in the ParticipantTable in the user's local timezone and we could use that same place to indicate the difference in timezones

For now, I noticed that I can display an abbreviation for a given timezone, so I'll go with that to reduce clutter

Reply:
What kind of abbreviation do you mean? If it's `UTC+xx` then it may be confusing for people, so I'd rather keep the repeated long `Europe/Zurich` etc

Reply:
Okay, I agree, abbreviation are not as cool as I thought 😄 . Sao Paulo timezone's abbreviation is just `(-03)`, so I agree that leaving the long repeated timezone is better

Reply:
yeah, let's keep it as-is then :)